ProjectSettings/ProjectVersion.txt

```diff
@@ -1 +1 @@
-m_EditorVersion: 2017.1.1f1
+m_EditorVersion: 5.6.1f1
```

Comment:
Please update your Unity version to 2017.2.0f3 (I will update the project version today).